index.html

```diff
@@ -1020,6 +1169,70 @@ <h2>
         </dd>
       </dl>
     </section>
+    <section data-dfn-for="GamepadTouch">
+      <h2>
+        <dfn>GamepadTouch</dfn> Interface
+      </h2>
+      <p>
+        This interface defines a single touch event on a gamepad device that
+        supports input. The event consists of a touch id that uniquely
+        identifies the touch point from the time the input medium (e.g. finger,
+        stylus, etc) makes contact with the touch device, up to the time the
+        input medium is no longer making contact with the touch device.
+      </p>
+      <pre class="idl">
+        [Exposed=Window, SecureContext]
+        interface GamepadTouch {
+          readonly attribute unsigned long touchId;
+          readonly attribute octet surfaceId;
```

Comment:
This seems confusing... wouldn't it be better is gamepad had an unordered touch surface map with named surfaces? Could that be a thing? 

```JS
const trackpad = gamepad.touchSufaces.get("trackpad"); 
```